src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -3272,6 +3280,7 @@ internal static bool IsSynthesizedAttributeOptional(WellKnownMember attributeMem
                 case WellKnownMember.System_STAThreadAttribute__ctor:
                 case WellKnownMember.System_Runtime_CompilerServices_AsyncStateMachineAttribute__ctor:
                 case WellKnownMember.System_Runtime_CompilerServices_IteratorStateMachineAttribute__ctor:
+                case WellKnownMember.System_Runtime_CompilerServices_IsByRefLikeAttribute__ctor:
```

Comment:
>case WellKnownMember.System_Runtime_CompilerServices_IsByRefLikeAttribute__ctor: [](start = 16, length = 80)

If this is a temporary code, it should be marked with a prototype comment. I am not sure if this is even used by the current implementation because SynthesizeIsByRefLikeAttribute explicitly passes true for isOptional.